cmd/createtree/main.go

```diff
@@ -151,24 +152,30 @@ func newPK(opts *createOpts) (*any.Any, error) {
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PKCS11ConfigFile":
+		if opts.pkcs11ConfigPath == "" {
+			return nil, errors.New("empty PKCS11 config file path")
+		}
+		return ptypes.MarshalAny(&keyspb.PKCS11ConfigFile{opts.pkcs11ConfigPath})
```

Comment:
I've given this a bit more thought, and was wondering whether there'd be a scenario where you'd want to use two different modules simultaneously in a deployment? If so, the module path ought to go in the `PKCS11Config` proto. Otherwise, it should be a field in a `PKCS11SignerFactory` that is set via a `NewPKCS11SignerFactory(modulePath)` func (and provided by a flag in the various `main.go` files). I don't think putting the module path directly into the `CreateTreeRequest` is the right place, nor should it be a parameter to the `SignerFactory.NewSigner()` method, since most `SignerFactory` implementations won't use it.